pom.xml

```diff
@@ -121,7 +121,7 @@ under the License.
 		<chill.version>0.7.6</chill.version>
 		<zookeeper.version>3.4.10</zookeeper.version>
 		<curator.version>2.12.0</curator.version>
-		<jackson.version>2.7.9</jackson.version>
+		<jackson.version>2.9.8</jackson.version>
```

Comment:
we're mixing things here. Let's open one PR that bumps the flink-shaded version and related dependencies, and migrate to hadoop separately.

Reply:
Agree, So, we merge this PR first, right?